test/apps/iodemo/io_demo.cc

```diff
@@ -671,9 +673,14 @@ class DemoServer : public P2pDemoCommon {
         recv_data(conn, *iov, msg->tr.sn, w);
     }
 
+    virtual void dispatch_connection_accepted(UcxConnection* conn) {\
+        ++_curr_state.active_conns;
+    }
+
     virtual void dispatch_connection_error(UcxConnection *conn) {
         LOG << "deleting connection with status "
             << ucs_status_string(conn->ucx_status());
+        --_curr_state.active_conns;
```

Comment:
can server be an initiator of disconnect?

Reply:
no